packages/etcd/src/index.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const EventEmitter = require('events');
+const { Etcd3 } = require('etcd3');
+
+class KeyvEtcd extends EventEmitter {
+	constructor(url, options) {
+		super();
+		this.ttlSupport = false;
```

Comment:
can we make this an option where you can set it too true? If ttlSupport is valid is there additional options needed to make it work?